pkgs/os-specific/linux/kernel/common-config.nix

```diff
@@ -698,8 +698,8 @@ let
       THRUSTMASTER_FF    = yes;
       ZEROPLUS_FF        = yes;
 
-      MODULE_COMPRESS    = yes;
-      MODULE_COMPRESS_XZ = yes;
+      MODULE_COMPRESS    = whenOlder "5.13" yes;
+      MODULE_COMPRESS_XZ = whenOlder "5.13" yes;
```

Comment:
why not keep the `MODULE_COMPRESS_XZ` / we still want to use xz compressed modules I think?